frontend/src/Explorer/View/Dashboard/Blocks.purs

```diff
@@ -53,3 +59,40 @@ emptyBlocksView message =
     P.div
         [ P.className CSS.blocksWaiting ]
         [ P.text message ]
+
+currentBlocks :: State -> CBlockEntries
+currentBlocks state =
+    if expanded
+    then slice minBlockIndex (minBlockIndex + maxBlockRows) blocks
+    else slice 0 minBlockRows blocks
+    where
+        blocks = unwrapLatestBlocks $ state ^. latestBlocks
+        expanded = state ^. dashboardBlocksExpanded
+        currentBlockPage = state ^. (dashboardViewState <<< dbViewBlockPagination)
+        minBlockIndex = (currentBlockPage - 1) * maxBlockRows
+
+blocksFooterView :: State -> P.Html Action
+blocksFooterView state =
+    if expanded then
+        paginationView { label: translate (I18nL.common <<< I18nL.cOf) $ lang'
+                        , currentPage: currentBlockPage
+                        , maxPage: (+) minPagination <<< flip (/) maxBlockRows $ length blocks
```

Comment:
Simpler is `(minPagination / maxBlockRows) + (length blocks)` if that's the goal.
This is getting repetitive, maybe it's time we build a simple `purescript-table` component?